Dockerfile

```diff
@@ -14,17 +14,27 @@ WORKDIR /headlamp
 
 RUN cd ./backend && go build -o ./server ./cmd/
 
-RUN cd ./frontend && npm install && npm run build
-
 # Create a plugins folder if none exists so we copy whatever exists to the
 # image.
+RUN mkdir -p ./.plugins
+
+# Backwards compatibility, Create the plugins folder if none exists.
 RUN mkdir -p ./plugins
 
+
+RUN cd ./frontend && npm install && npm run build
+
 FROM $IMAGE_BASE
 
 COPY --from=base-build /headlamp/backend/server /headlamp/server
 COPY --from=base-build /headlamp/frontend/build /headlamp/frontend
+# Backwards compatibility, "./plugins" was the old name.
 COPY --from=base-build /headlamp/plugins /headlamp/plugins
+COPY --from=base-build /headlamp/.plugins /headlamp/.plugins
+
+# Backwards compatibility, copy any plugins found inside "./plugins" into ".plugins".
+# They should match plugins/MyFolder/main.js, otherwise they are not copied.
+RUN find plugins -maxdepth 2 -type f  | grep -i main.js$ | xargs -i dirname {} | xargs -i cp -a {} .plugins/
```

Comment:
Unless my tests didn't work (I tested with the lokomotive-web-ui plugins), I ended up with /headlamp/.plugins empty.